### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by kycarr**

### CODE REVIEW

**Review by beatthat — CHANGES_REQUESTED** [review]
this repo now actually contains two services--`opentutor-classifier` and `opentutor-classifier-api`. 

Each of the above has it's own set of tests and deploys a docker image

**Review by beatthat — CHANGES_REQUESTED** [review]
looks great. Can we just add newlines at the end of the 3 new yaml files?

File: .github/workflows/test-commit.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - name: Run format test
+        run: cd opentutor_classifier_api && make test-format
+      - name: Run lint test
+        run: cd opentutor_classifier_api && make test-lint
+      - name: Run types test
+        run: cd opentutor_classifier_api && make test-types
+      - name: Run license test
+        run: cd opentutor_classifier_api && make test-license
+      - name: Run tests
+        run: cd opentutor_classifier_api && make test
```
Comment (beatthat) [inline .github/workflows/test-commit.yml:41]: these 3 new yaml files still missing the the newline at the EOF

**Review by beatthat — CHANGES_REQUESTED** [review]

File: .github/workflows/test-commit.yml (hunk 1 above)
Comment (kycarr) [reply .github/workflows/test-commit.yml:41]: I see them in the code though? I think they just don't show in github.

**Review by beatthat — APPROVED** [review]

**Pull request merged**